### ISSUE TITLE
Reply to Email Message has incorrect date on original email

### ISSUE THREAD

**Issue by ajitsinghsethi** [OfficeDev/ews-java-api#591]
I'm sending a reply back to an email message, when I receive the reply, the Sent date is incorrect on the actual email. 
Actual date Sent- Tue 1/3/2017 3:58 PM
In the reply message date sent  - Tuesday, January 3, 2017 8:58:51 PM

here is my reply code-

`PropertySet propertySet = new PropertySet(BasePropertySet.IdOnly,
            EmailMessageSchema.From, EmailMessageSchema.CcRecipients,
            EmailMessageSchema.Subject, EmailMessageSchema.Body,
            EmailMessageSchema.Sender, EmailMessageSchema.DateTimeReceived,
            EmailMessageSchema.Attachments);

    propertySet.setRequestedBodyType(BodyType.HTML);

    String itemId = emailMessage.getId().toString();
    EmailMessage message = EmailMessage.bind(service, new ItemId(itemId), propertySet);
    //message.getIsTimeZoneHeaderRequired(true);
    //getESTTimeZone(service);
    MessageBody errorMessage = new MessageBody();
    errorMessage.setBodyType(BodyType.HTML);
    errorMessage.setText(returnMessage);
    message.reply(errorMessage, false);   //false means do not reply all`

is this an existing issue with EWS java 2.0?

**Comment by davster** [comment]
Date sent is UTC there.  EST is 5 hours behind UTC.

**Comment by ajitsinghsethi** [comment]
Yes, We use EST and actual Date Sent is EST, but after I send the reply it changes to UTC date, How can I keep the same date or configure my reply message or the ExchangeService to use EST.? 

I tried using OnSerializeCustomSoapHeaders with no luck,
[https://blogs.msdn.microsoft.com/emeamsgdev/2014/04/23/ews-missing-soap-headers-when-using-the-ews-managed-api/](url)

here is how I tried adding the customSoapHeader-

`private static void getSerializeCustomSoapHeaders(ExchangeService service) throws Exception {
		
		//SerializeCustomSoapHeaders
		List<ICustomXmlSerialization> serialHeader = new ArrayList<ICustomXmlSerialization>();
		
		ICustomXmlSerialization element = new ICustomXmlSerialization() {
			
			@Override
			public void CustomXmlSerialization(XMLStreamWriter xmlstreamwriter) {
				try {
					//writer.
					//xmlstreamwriter.writeCData(System.lineSeparator() + "    <t:TimeZoneContext><t:TimeZoneDefinition Id=\"Eastern Standard Time\"/></t:TimeZoneContext>" + System.lineSeparator());
				} catch(Exception e) {
					e.printStackTrace();
				}
			}
		};
		
		OutputStream stream = System.out;

        XMLOutputFactory xof = XMLOutputFactory.newFactory();
        XMLStreamWriter xsw = xof.createXMLStreamWriter(stream);

        xsw.writeStartDocument();
        xsw.writeStartElement("t:TimeZoneContext");

        /* Following line is very important, without it unescaped data 
           will appear inside the <bar> tag. */
        xsw.writeCharacters("");
        xsw.flush();

        OutputStreamWriter osw = new OutputStreamWriter(stream);
        osw.write("<t:TimeZoneDefinition Id=\"Eastern Standard Time\"/>");
        //xsw.writeStartElement("</t:TimeZoneContext>");
        osw.flush();

        xsw.writeEndElement();
        
        xsw.writeEndDocument();
        xsw.close();
		//XMLStreamWriter xmlstreamwriter = XMLOutputFactory.newInstance().createXMLStreamWriter(out);
		//xmlstreamwriter.writeCData(System.lineSeparator() + "    <t:TimeZoneContext><t:TimeZoneDefinition Id=\"Eastern Standard Time\"/></t:TimeZoneContext>" + System.lineSeparator());
		element.CustomXmlSerialization(xsw);
		serialHeader.add(element);
		
		service.setOnSerializeCustomSoapHeaders(serialHeader);
		
		/*List<ICustomXmlSerialization> response = service.getOnSerializeCustomSoapHeaders();
		for (Iterator iterator = response.iterator(); iterator.hasNext();) {
			ICustomXmlSerialization iCustomXmlSerialization = (ICustomXmlSerialization) iterator.next();
			System.out.println(iCustomXmlSerialization);
		}*/
		
		//return null;
	}
is this how it should be added? `

there are other options where they say to change the source code.
[http://stackoverflow.com/questions/28607832/ews-managed-api-timezonedefinition-header-not-sent-when-forwarding](url)
